### ISSUE TITLE
Macless-Haystack

### ISSUE THREAD

**Issue by omarkhali** [malmeloo/hass-FindMy#2]
Hi I have a Macless-Haystack server running on my network and I see my devices in the web and I want to see my devices in the mab Home Assistant
I have installed FindMy - Home Assistant integration
Add entity
FindMy device
I added Private key and name
And I did not find anything
Should I add my account
Or is it lost anisette server

**Comment by malmeloo** [comment]
Hi! Yes, you need to set up at least one account with an Anisette server. When setting up an account, check the 'advanced options' dropdown menu to enter the URL of your anisette server; you can use a public one, but I highly recommend using a private server (such as [anisette-v3-server](https://github.com/Dadoum/anisette-v3-server)) for better stability.

Fetching reports might take a little bit of time (~30 minutes for now). If you need the locations to update faster, you can add more accounts to reduce the delay to `30 / n` minutes. Feel free to reach out if you have more questions 🙂

**Comment by omarkhali** [comment]
So I no longer need Macless-Haystack server
I just need anisette server

**Comment by malmeloo** [comment]
For this integration, no, you don't need the macless-haystack server. My intent is to phase out the anisette server requirement as well at some point, but for now you will need still need it.

**Comment by omarkhali** [comment]
Is there a problem if I leave the Macless-Haystack server running too?

**Comment by malmeloo** [comment]
Nope, shouldn't be an issue. The integration will not use it.

**Comment by omarkhali** [comment]
Thank you very much

**Comment by omarkhali** [comment]
A question out of curiosity, is it possible to make anisette-v3-server home assistant add ons because I searched everywhere and did not find any?

**Comment by omarkhali** [comment]
Another question: Is it possible to add more than one account to the same anisette-v3-server or does each account have anisette-v3-server?

**Comment by malmeloo** [comment]
Technically it would be possible to run the anisette server as an addon, but the effort to do that would be better spent directly implementing anisette in the underlying library. This way you would not even have to set up your own server in the first place. It's already on the roadmap :)

And yes, you can run multiple accounts on the same anisette server just fine.

**Comment by omarkhali** [comment]
Thank you for your responses and sorry for bothering you.

**Comment by malmeloo** [comment]
No worries, you're not bothering me at all. Feel free to ask :-)

**Comment by cyber-vi-king** [comment]
Hi, I just found this project and I have a question about the basics. 

The anisette server doesn’t need any config? So if I run your project and the anisette docker container, then I can add apple accounts via your integration and the anisette server in docker just does its magic behind the scenes?

So:
- run anisette docker
- install your integration 
- add apple account(s)
- add openhaystack tags
- success…

Or am I forgetting anything?

**Comment by malmeloo** [comment]
Yes, but make sure to mount the volume (according to anisette-v3-server's instructions) somewhere. If you don't, the server's config will not be retained and the integration will keep losing its login session. Then when adding an apple account in home assistant, specify the url to the server in advanced settings.

Closing this issue because I think the original question should now be answered.
